.github/workflows/propagate-doc-upwards.yml

```diff
@@ -53,37 +53,15 @@ jobs:
 
       - name: Send message to Slack channel
         if: failure() && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.slack-notifications == 'true') )
-        uses: slackapi/slack-github-action@v1.23.0
+        uses: bonitasoft/actions/packages/notify-slack@v2.2
```

Comment:
**suggestion (if-minor):** we could use v2 to limit maintenance. We are maintaining the action so we can trust our-self to preserve backwards compatibility for the v2 version 😸 

```suggestion
        uses: bonitasoft/actions/packages/notify-slack@v2
```